### PR TITLE
Correct Cinder Backend Name for Ceph

### PR DESCRIPTION
**Pull request by mgaubert**
+ Ceph Ansible create volumes rdb pool for storing cinder volumes
+ Use the same name for Cinder backend

### CODE REVIEW

File: etc/openstack_deploy/templates/openstack_user_config.yml.j2 [hunk 1]
```diff
@@ -503,7 +503,7 @@ storage_hosts:
           volume_group: cinder-volumes
           volume_driver: cinder.volume.drivers.rbd.RBDDriver
           volume_backend_name: rbd
-          rbd_pool: cinder_volumes
+          rbd_pool: volumes
```
Comment (ppetit) [inline etc/openstack_deploy/templates/openstack_user_config.yml.j2]: Why do you want to change rbd_pool name cinder_volumes by volumes? IMO cinder_volumes is better explanatory and should not be changed.

File: etc/openstack_deploy/templates/user_variables.yml.j2 [hunk 2]
```diff
@@ -194,7 +194,7 @@ cinder_cinder_conf_overrides:
 {% endif %}
 
 {% if 'ceph' in onpc_services_list %}
-cinder_default_volume_type: rbddriver
+cinder_default_volume_type: rbd
```
Comment (ppetit) [inline etc/openstack_deploy/templates/user_variables.yml.j2]: Good catch!

File: etc/openstack_deploy/templates/openstack_user_config.yml.j2 (hunk 1 above)
Comment (mgaubert) [reply etc/openstack_deploy/templates/openstack_user_config.yml.j2]: ceph-ansible configures it this way (https://github.com/ceph/ceph-ansible/blob/04ec1ad3cc0d4b60458a8b14dcc0b640e80f9eef/tests/functional/podman/group_vars/all)

Comment (ppetit) [reply etc/openstack_deploy/templates/openstack_user_config.yml.j2]: It's a just a name example. IMO, 'cinder-volumes' as opposed to just 'volumes' is more self-explanatory because you can tell by the name that the pool is about cinder and not something else. I prefer it this way.

Comment (mgaubert) [reply etc/openstack_deploy/templates/openstack_user_config.yml.j2]: The problem is that cinder wont create the rdb pool and if you want to use cinder_volumes as the rdb pool name, you must also change the ceph configuration showed in the link.

Comment (ppetit) [reply etc/openstack_deploy/templates/openstack_user_config.yml.j2]: Okay but that's not a problem of name. It used to work... I think the problem is more about that volume_group: cinder-volumes and rbd_pool: cinder_volumes no  longer match. There has been a clunky change at some point there :(  Check https://docs.openstack.org/openstack-ansible-os_cinder/latest/configure-cinder.html#configuring-cinder-to-use-ceph
And working example at https://www.openstackfaq.com/openstack-ansible-ceph/
In summary, I think the correction is to define rbd_pool as cinder**-**volumes (with an hyphen)
